Aspire Budgeting/Views/CardView/CardListView.swift

```diff
@@ -7,16 +7,16 @@ import SwiftUI
 
 struct CardListView: View {
 
-  let cardViewItems: [CardView.CardViewItem]
+  let cardViewItems: [BaseCardView.CardViewItem]
 
   var body: some View {
     GeometryReader {g in
       ScrollView {
         VStack {
           ForEach(0..<self.cardViewItems.count) { idx in
             GeometryReader { geo in
-              CardView(colorInfo: CardListView.colorInfos[idx % CardListView.colorInfos.count],
-                       cardViewItem: self.cardViewItems[idx],
+              BaseCardView(colorInfo: CardListView.colorInfos[idx % CardListView.colorInfos.count],
+//                       cardViewItem: self.cardViewItems[idx],
                        minY: g.frame(in: .global).minY,
                        curY: geo.frame(in: .global).minY)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/6c344c52764dd09f4d37a8501914d1161cb941a9/Aspire Budgeting/Views/CardView/CardListView.swift#L21"></span>Function parameters should be aligned vertically if they’re in multiple lines in a method call.<br />
<code>vertical_parameter_alignment_on_call</code> <code>CardListView.swift:21</code></td>
    </tr>
  </tbody>
</table>